lib/pg_search/features/tsearch.rb

```diff
@@ -94,7 +94,7 @@ def ts_headline_option_value(value)
         end
       end
 
-      DISALLOWED_TSQUERY_CHARACTERS = /['?\\:‘’]/.freeze
+      DISALLOWED_TSQUERY_CHARACTERS = /['?\\:‘’ʻʼ]/.freeze
```

Comment:
`＇`  U+FF07	FULLWIDTH apostrophe is another character that gets transformed to `'` by `unaccent`